### DISCUSSION TITLE
Invalid conversion mode customary

### DISCUSSION THREAD

**Discussion by TGSnedecor** [merbanan/rtl_433#2358] [Q&A] [answered]
When I run from the command line, I enter -C customary , which seems to work fine.

However, I want to put my command line switches into a rtl_433.conf

When I add 

convert customary to the .conf file, rtl_433 fails to start with the error:

"Invalid conversion mode customary"

Can someone help me correct what I'm doing wrong?  Thanks!

**Comment by zuckschwerdt** [comment, marked as answer]
You likely added trailing whitespace or quoting, which is not supported :/
